# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1458012152 14400
#      Mon Mar 14 23:22:32 2016 -0400
# Node ID 2868abbc88eb9d01ac8f65194b01453e7e602a54
# Parent  65e24ba4ca2d743e24817fc2622dc402c33363a5
eliminate unused configuration variable SHLLINKEXT

* common.mk, configure.ac (SHLLINKEXT): Delete unused variable.
* subst-config-vals.in.sh, subst-cross-config-vals.in.sh:
Don't substitute it.

diff --git a/build-aux/common.mk b/build-aux/common.mk
--- a/build-aux/common.mk
+++ b/build-aux/common.mk
@@ -9,20 +9,16 @@ version := ${OCTAVE_VERSION}
 api_version := ${OCTAVE_API_VERSION}
 
 ## AM_LIBTOOLFLAGS = --silent
 
 AM_LFLAGS = @LFLAGS@
 
 AM_YFLAGS = -dv
 
-# FIXME: This seems unnecessary as of 3/10/2016.
-# Commenting out definition and re-configuring made no change to Makefile
-SHLLINKEXT =
-
 # Fortran compiler flags.
 
 AM_FFLAGS = @FFLAGS@
 
 # C compiler flags.
 
 AM_CFLAGS = ${XTRA_CFLAGS}
 
diff --git a/build-aux/subst-config-vals.in.sh b/build-aux/subst-config-vals.in.sh
--- a/build-aux/subst-config-vals.in.sh
+++ b/build-aux/subst-config-vals.in.sh
@@ -147,17 +147,16 @@ QT_CPPFLAGS="@QT_CPPFLAGS@"
 QT_LDFLAGS="@QT_LDFLAGS@"
 QT_LIBS="@QT_LIBS@"
 RANLIB="@RANLIB@"
 RDYNAMIC_FLAG="@RDYNAMIC_FLAG@"
 READLINE_LIBS="@READLINE_LIBS@"
 SED="@SED@"
 SHARED_LIBS="@SHARED_LIBS@"
 SHLEXT="@SHLEXT@"
-SHLLINKEXT="@SHLLINKEXT@"
 SHLEXT_VER="@SHLEXT_VER@"
 SH_LD="@SH_LD@"
 SH_LDFLAGS="@SH_LDFLAGS@"
 STATIC_LIBS="@STATIC_LIBS@"
 TERM_LIBS="@TERM_LIBS@"
 UMFPACK_CPPFLAGS="@UMFPACK_CPPFLAGS@"
 UMFPACK_LDFLAGS="@UMFPACK_LDFLAGS@"
 UMFPACK_LIBS="@UMFPACK_LIBS@"
@@ -304,17 +303,16 @@ config_opts="@config_opts@"
   -e "s|%OCTAVE_CONF_QT_LDFLAGS%|\"${QT_LDFLAGS}\"|" \
   -e "s|%OCTAVE_CONF_QT_LIBS%|\"${QT_LIBS}\"|" \
   -e "s|%OCTAVE_CONF_RANLIB%|\"${RANLIB}\"|" \
   -e "s|%OCTAVE_CONF_RDYNAMIC_FLAG%|\"${RDYNAMIC_FLAG}\"|" \
   -e "s|%OCTAVE_CONF_READLINE_LIBS%|\"${READLINE_LIBS}\"|" \
   -e "s|%OCTAVE_CONF_SED%|\"${SED}\"|" \
   -e "s|%OCTAVE_CONF_SHARED_LIBS%|\"${SHARED_LIBS}\"|" \
   -e "s|%OCTAVE_CONF_SHLEXT%|\"${SHLEXT}\"|" \
-  -e "s|%OCTAVE_CONF_SHLLINKEXT%|\"${SHLLINKEXT}\"|" \
   -e "s|%OCTAVE_CONF_SHLEXT_VER%|\"${SHLEXT_VER}\"|" \
   -e "s|%OCTAVE_CONF_SH_LD%|\"${SH_LD}\"|" \
   -e "s|%OCTAVE_CONF_SH_LDFLAGS%|\"${SH_LDFLAGS}\"|" \
   -e "s|%OCTAVE_CONF_STATIC_LIBS%|\"${STATIC_LIBS}\"|" \
   -e "s|%OCTAVE_CONF_TERM_LIBS%|\"${TERM_LIBS}\"|" \
   -e "s|%OCTAVE_CONF_UMFPACK_CPPFLAGS%|\"${UMFPACK_CPPFLAGS}\"|" \
   -e "s|%OCTAVE_CONF_UMFPACK_LDFLAGS%|\"${UMFPACK_LDFLAGS}\"|" \
   -e "s|%OCTAVE_CONF_UMFPACK_LIBS%|\"${UMFPACK_LIBS}\"|" \
diff --git a/build-aux/subst-cross-config-vals.in.sh b/build-aux/subst-cross-config-vals.in.sh
--- a/build-aux/subst-cross-config-vals.in.sh
+++ b/build-aux/subst-cross-config-vals.in.sh
@@ -151,17 +151,16 @@ QT_CPPFLAGS="@QT_CPPFLAGS@"
 QT_LDFLAGS="@QT_LDFLAGS@"
 QT_LIBS="@QT_LIBS@"
 RANLIB="@RANLIB@"
 RDYNAMIC_FLAG="@RDYNAMIC_FLAG@"
 READLINE_LIBS="@READLINE_LIBS@"
 SED="@SED@"
 SHARED_LIBS="@SHARED_LIBS@"
 SHLEXT="@SHLEXT@"
-SHLLINKEXT="@SHLLINKEXT@"
 SHLEXT_VER="@SHLEXT_VER@"
 SH_LD="@SH_LD@"
 SH_LDFLAGS="@SH_LDFLAGS@"
 STATIC_LIBS="@STATIC_LIBS@"
 TERM_LIBS="@TERM_LIBS@"
 UMFPACK_CPPFLAGS="@UMFPACK_CPPFLAGS@"
 UMFPACK_LDFLAGS="@UMFPACK_LDFLAGS@"
 UMFPACK_LIBS="@UMFPACK_LIBS@"
@@ -308,17 +307,16 @@ config_opts="@config_opts@"
     -e "s|%OCTAVE_CONF_QT_LDFLAGS%|\"${QT_LDFLAGS}\"|" \
     -e "s|%OCTAVE_CONF_QT_LIBS%|\"${QT_LIBS}\"|" \
     -e "s|%OCTAVE_CONF_RANLIB%|\"${RANLIB}\"|" \
     -e "s|%OCTAVE_CONF_RDYNAMIC_FLAG%|\"${RDYNAMIC_FLAG}\"|" \
     -e "s|%OCTAVE_CONF_READLINE_LIBS%|\"${READLINE_LIBS}\"|" \
     -e "s|%OCTAVE_CONF_SED%|\"${SED}\"|" \
     -e "s|%OCTAVE_CONF_SHARED_LIBS%|\"${SHARED_LIBS}\"|" \
     -e "s|%OCTAVE_CONF_SHLEXT%|\"${SHLEXT}\"|" \
-    -e "s|%OCTAVE_CONF_SHLLINKEXT%|\"${SHLLINKEXT}\"|" \
     -e "s|%OCTAVE_CONF_SHLEXT_VER%|\"${SHLEXT_VER}\"|" \
     -e "s|%OCTAVE_CONF_SH_LD%|\"${SH_LD}\"|" \
     -e "s|%OCTAVE_CONF_SH_LDFLAGS%|\"${SH_LDFLAGS}\"|" \
     -e "s|%OCTAVE_CONF_STATIC_LIBS%|\"${STATIC_LIBS}\"|" \
     -e "s|%OCTAVE_CONF_TERM_LIBS%|\"${TERM_LIBS}\"|" \
     -e "s|%OCTAVE_CONF_UMFPACK_CPPFLAGS%|\"${UMFPACK_CPPFLAGS}\"|" \
     -e "s|%OCTAVE_CONF_UMFPACK_LDFLAGS%|\"${UMFPACK_LDFLAGS}\"|" \
     -e "s|%OCTAVE_CONF_UMFPACK_LIBS%|\"${UMFPACK_LIBS}\"|" \
diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -1999,17 +1999,16 @@ CPICFLAG=-fPIC
 CXXPICFLAG=-fPIC
 FPICFLAG=-fPIC
 SHLEXT=so
 SHLLIB="${SHLEXT}"
 SHLBIN=
 SHLEXT_VER="${SHLEXT}.${version}"
 SHLLIB_VER="${SHLLIB}.${version}"
 SHLBIN_VER="${SHLBIN}.${version}"
-SHLLINKEXT=
 LIBPRE=lib
 SHLPRE=lib
 SHLLIBPRE=lib
 SHLBINPRE=lib
 SH_LD="${CXX}"
 SH_LDFLAGS=-shared
 DL_LD="${SH_LD}"
 DL_LDFLAGS="${SH_LDFLAGS}"
@@ -2217,17 +2216,16 @@ AC_MSG_NOTICE([defining FPICFLAG to be $
 AC_MSG_NOTICE([defining CPICFLAG to be $CPICFLAG])
 AC_MSG_NOTICE([defining CXXPICFLAG to be $CXXPICFLAG])
 AC_MSG_NOTICE([defining SHLEXT to be $SHLEXT])
 AC_MSG_NOTICE([defining SHLLIB to be $SHLLIB])
 AC_MSG_NOTICE([defining SHLBIN to be $SHLBIN])
 AC_MSG_NOTICE([defining SHLEXT_VER to be $SHLEXT_VER])
 AC_MSG_NOTICE([defining SHLLIB_VER to be $SHLLIB_VER])
 AC_MSG_NOTICE([defining SHLBIN_VER to be $SHLBIN_VER])
-AC_MSG_NOTICE([defining SHLLINKEXT to be $SHLLINKEXT])
 AC_MSG_NOTICE([defining LIBPRE to be $LIBPRE])
 AC_MSG_NOTICE([defining SHLPRE to be $SHLPRE])
 AC_MSG_NOTICE([defining SHLLIBPRE to be $SHLLIBPRE])
 AC_MSG_NOTICE([defining SHLBINPRE to be $SHLBINPRE])
 AC_MSG_NOTICE([defining SH_LD to be $SH_LD])
 AC_MSG_NOTICE([defining SH_LDFLAGS to be $SH_LDFLAGS])
 AC_MSG_NOTICE([defining DL_LD to be $DL_LD])
 AC_MSG_NOTICE([defining DL_LDFLAGS to be $DL_LDFLAGS])
@@ -2246,17 +2244,16 @@ AC_SUBST(FPICFLAG)
 AC_SUBST(CPICFLAG)
 AC_SUBST(CXXPICFLAG)
 AC_SUBST(SHLEXT)
 AC_SUBST(SHLLIB)
 AC_SUBST(SHLBIN)
 AC_SUBST(SHLEXT_VER)
 AC_SUBST(SHLLIB_VER)
 AC_SUBST(SHLBIN_VER)
-AC_SUBST(SHLLINKEXT)
 AC_SUBST(LIBPRE)
 AC_SUBST(SHLPRE)
 AC_SUBST(SHLLIBPRE)
 AC_SUBST(SHLBINPRE)
 AC_SUBST(SH_LD)
 AC_SUBST(SH_LDFLAGS)
 AC_SUBST(DL_LD)
 AC_SUBST(DL_LDFLAGS)
@@ -2364,18 +2361,18 @@ fi
 if test $ENABLE_DYNAMIC_LINKING = yes; then
   AC_DEFINE(ENABLE_DYNAMIC_LINKING, 1, [Define to 1 if using dynamic linking.])
 fi
 
 AM_CONDITIONAL([AMCOND_ENABLE_DYNAMIC_LINKING],
   [test $ENABLE_DYNAMIC_LINKING = yes])
 
 if test $SHARED_LIBS = yes; then
-  LIBOCTINTERP="-loctinterp$SHLLINKEXT"
-  LIBOCTAVE="-loctave$SHLLINKEXT"
+  LIBOCTINTERP="-loctinterp"
+  LIBOCTAVE="-loctave"
 else
   LIBOCTINTERP="${top_builddir}/libinterp/liboctinterp.${LIBEXT}"
   LIBOCTAVE="${top_builddir}/liboctave/liboctave.${LIBEXT}"
 fi
 
 AC_SUBST(LD_CXX)
 AC_SUBST(RDYNAMIC_FLAG)
 AC_SUBST(ENABLE_DYNAMIC_LINKING)
